sdk/src/types/block/slot/index.rs

```diff
@@ -43,13 +43,22 @@ impl SlotIndex {
     }
 
     /// Gets the slot index of a unix timestamp.
+    /// Slots are counted starting from `1` with `0` being reserved for times before the genesis.
```

Comment:
This sentence got updated in https://github.com/iotaledger/tips-draft/pull/20/commits/e95818e621e3edbcc6693b110024781bf5631a3f

Reply:
Nothing wrong, just wasn't sure if you maybe want to add the other part, but also fine without it